src/test/java/me/ajaja/module/footprint/domain/FootprintTest.java

```diff
@@ -0,0 +1,56 @@
+package me.ajaja.module.footprint.domain;
+
+import static org.assertj.core.api.Assertions.assertThatNoException;
+
+import java.sql.Blob;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import me.ajaja.common.support.MonkeySupport;
+import me.ajaja.module.footprint.dto.FootprintParam;
+
+import com.navercorp.fixturemonkey.FixtureMonkey;
+import com.navercorp.fixturemonkey.api.introspector.ConstructorPropertiesArbitraryIntrospector;
+import com.navercorp.fixturemonkey.jakarta.validation.plugin.JakartaValidationPlugin;
+
+class FootprintTest extends MonkeySupport {
+	private final FixtureMonkey fixtureMonkey = FixtureMonkey.builder()
+		.objectIntrospector(ConstructorPropertiesArbitraryIntrospector.INSTANCE)
+		.plugin(new JakartaValidationPlugin())
+		.defaultNotNull(true)
+		.build();
+
+	@Nested
+	@DisplayName("유형별 발자취 도메인 생성을 성공한다")
+	class CreateFootprintTest {
+		@Test
+		@DisplayName("자유 형식 발자취 도메인 생성을 성공한다.")
+		public void createFreeFootprint_Success_WithNoException() {
+			assertThatNoException().isThrownBy(
+				() -> {
+					FootprintParam.Create createParam = fixtureMonkey.giveMeOne(FootprintParam.Create.class);
+					Blob content = fixtureMonkey.giveMeOne(Blob.class);
+
+					FreeFootprint freeFootprint = new FreeFootprint(createParam, content);
+				}
+			);
+		}
+
+		@Test
+		@DisplayName("KPT 형식 발자취 도메인 생성을 성공한다.")
+		public void createKptFootprint_Success_WithNoException() {
+			assertThatNoException().isThrownBy(
+				() -> {
+					FootprintParam.Create createParam = fixtureMonkey.giveMeOne(FootprintParam.Create.class);
+					Blob keepContent = fixtureMonkey.giveMeOne(Blob.class);
+					Blob problemContent = fixtureMonkey.giveMeOne(Blob.class);
+					Blob tryContent = fixtureMonkey.giveMeOne(Blob.class);
+
```

Comment:
나중에 blob을 안 쓰시겠지만 이런식으로도 사용할 수 있습니다 ㅎㅎ
```suggestion
					var blobBuilder = sut.giveMeBuilder(Blob.class);
					Blob keepContent = blobBuilder.sample();
					Blob problemContent = blobBuilder.sample();
					Blob tryContent = blobBuilder.sample();
```

Reply:
빌더가 있었군요!!! 픽스처 몽키랑 더 친해져야겠네요!!